api/test/integration/import-data/xlsx-uploads/.~lock.base-dataset.xlsx#

```diff
@@ -0,0 +1 @@
+,yulia,omen,20.01.2022 17:46,file:///home/yulia/snap/libreoffice/239/.config/libreoffice/4;
```

Comment:
![image](https://user-images.githubusercontent.com/973964/151137611-824201c7-494c-4a5a-a3a3-549de0ae4db8.png)

Reply:
cleaned the mess, sorry 😬